File: ANSv4/CatalogoLibraryVB/_recibos/AplicacionItem.vb
Option Explicit On

Public Class AplicacionItem

    ' Define como se llama este modulo para el control de errores
    Private Const m_sMODULENAME_ As String = "clsAplicacionItem"

    'variables locales para almacenar los valores de las propiedades
    Private mvarConcepto As String 'copia local
    Private mvarImporte As Single 'copia local
    Private mvarObservaciones As String 'copia local

    Public Property Observaciones() As String
        Set(ByVal value As String)
            mvarObservaciones = value
        End Set
        Get
            Return mvarObservaciones
        End Get
    End Property

    Public Property Importe() As Single
        Set(ByVal value As Single)
            mvarImporte = value
        End Set
        Get
            Return mvarImporte
        End Get
    End Property

    Public Property Concepto() As String
        Set(ByVal value As String)        
            mvarConcepto = value
        End Set
        Get
            Return mvarConcepto
        End Get
    End Property

End Class

File: ANSv4/Catalogo4/_recibos/AplicacionItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Catalogo._recibos
{
    public class AplicacionItem
    {

        //variables locales para almacenar los valores de las propiedades
        //copia local
        private string mvarConcepto;
        //copia local
        private float mvarImporte;
        //copia local
        private string mvarObservaciones;

        public string Observaciones
        {
            get { return mvarObservaciones; }
            set { mvarObservaciones = value; }
        }

        public float Importe
        {
            get { return mvarImporte; }
            set { mvarImporte = value; }
        }

        public string Concepto
        {
            get { return mvarConcepto; }
            set { mvarConcepto = value; }
        }

    }
}
